docs/css/doc.css

```diff
@@ -97,6 +97,16 @@
   display: block;
 }
 
+/*Grids*/
+.container_component .grid {
+  width: 100%;
+}
+
+.container_component .grid > * {
+  border: 1px solid var(--border);
+  min-height: 100px;
```

Comment:
could be converted to relative units

Reply:
thanks for pointing that out. Will do that.